### PR TITLE
Add test case for recover affect from deleting

### PR DESCRIPTION
**Pull request by huiwangredhat**
# [OSIDB-1699] [Affect-recover]

### CODE REVIEW

File: features/steps/flaw_detail.py [hunk 1]
```diff
@@ -551,8 +551,10 @@ def step_impl(context):
     flaw_detail_page.check_value_exist(context.ps_component)
     context.browser.quit()
 
-@when("I delete an affect of the flaw")
-def step_impl(context):
+def click_affect_delete_btn(context):
```
Comment (RichardXuan) [inline features/steps/flaw_detail.py:554]: Better put this function into `FlawDetailPage` class

File: features/steps/flaw_detail.py [hunk 2]
```diff
@@ -573,4 +581,18 @@ def step_impl(context):
...
+    context.ps_component = click_affect_delete_btn(context)
+    # Click the delete button of the affect
+    flaw_detail_page = FlawDetailPage(context.browser)
+    flaw_detail_page.click_button_with_js('affectRecoverBtn')
+
+@then("I could 'recover' the affect that I tried to delete above")
+def step_impl(context):
+    go_to_specific_flaw_detail_page(context.browser)
+    flaw_detail_page = FlawDetailPage(context.browser)
+    flaw_detail_page.check_value_exist(context.ps_component)
```
Comment (RichardXuan) [inline features/steps/flaw_detail.py:597]: Is it sufficient to only check one field value.

File: features/steps/flaw_detail.py (hunk 1 above)
Comment (huiwangredhat) [reply features/steps/flaw_detail.py:552]: Updated.

File: features/steps/flaw_detail.py (hunk 2 above)
Comment (huiwangredhat) [reply features/steps/flaw_detail.py]: Updated to check the module and component.

**Review by RichardXuan — APPROVED** [review]
LGTM

**Pull request merged**